src/rai_bench/rai_bench/tool_calling_agent/predefined/navigation_tasks.py

```
@@ -0,0 +1,111 @@
+    },
+)
+######### VALIDATORS #########################################################################################
+start_navigate_action_ord_val = OrderedCallsValidator(
+    subtasks=[start_nav_action_subtask]
+)
+start_spin_action_ord_val = OrderedCallsValidator(subtasks=[start_spin_action_subtask])
+move_ahead_ord_val = OrderedCallsValidator(subtasks=[start_move_front_action_subtask])
+
+
+def get_navigation_tasks(
+    extra_tool_calls: List[int] = [0],
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)